CHANGES.rst

```diff
@@ -4,7 +4,10 @@
 4.6.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Fix a crash in the test suite under a 32-bit CPython on certain
+  platforms. See `issue 137
+  <https://github.com/zopefoundation/persistent/issues/137>`_. Fix by
+  `Jerry James <https://github.com/jamesjer>`_.
```

Comment:
I got the impression that while the crash happened to be triggered in the test suite, it was actually a real bug and could just as well have crashed a real application?

Reply:
Also, s/certain platforms/32-bit platforms/ ?

Reply:
It *could*, but it's unlikely. One would have to opt-in to using the pure-Python implementation on CPython (which is incredibly slow).